src/utils.ts

```diff
@@ -203,3 +203,12 @@ const urlsWebApp: any = [
 export function getCurrentPageForUrl(currentUrl: string): any {
   return urlsWebApp.find((item: any) => item.url === currentUrl);
 }
+
+export function findUrlInRetryList(currentUrl: any): any {
+  const urls = [
+    'WebAppPublic/Login',
+    'WebAppPublic/CreateUser',
+    '/WebAppPublic/SendResetPasswordEmail',
+  ];
+  return urls.find((item: string) => item === currentUrl);
+}
```

Comment:
I think that these URLs are very tied to legacy Doppler, what do you think about keep it inside doppler-legacy-client.ts?

Reply:
fixed